cwatm/hydrological_modules/water_quality/waterquality_main.py

```diff
@@ -57,27 +59,38 @@ def initial(self):
         
         # dynamic -> load inputs and ground cover adjustment
         # dynamic_soil, dynamic_...
-        
-        # check if water quality is included
+        """
+
+        # check settings file, if water quality should be included
         self.var.includeWaterQuality =  False
         if 'includeWaterQuality' in option:
             self.var.includeWaterQuality =  checkOption('includeWaterQuality')
 
         if self.var.includeWaterQuality:
             # load sub-modules
-             
             self.var.includePhosphorus = False
             if 'includePhosphorus' in binding:
                 self.var.includePhosphorus = returnBool('includePhosphorus')
+
+            self.var.includeErosed = False
+            if 'includeErosed' in binding:
+                self.var.includeErosed = returnBool('includeErosed')
             
             
             # run initial sub-modules
             
             if self.var.includePhosphorus:
                 self.waterquality_p.initial()
+
+            if self.var.includeErosed:
+                self.erosed.initial()
         
             
     def dynamic(self):
-        print("wq dyn.")
-    
-       
\ No newline at end of file
+
+        if self.var.includeWaterQuality:
```

Comment:
It is not necessarily the place to run dynamic. I guess you might want to sync it with the soil module